features/org.wso2.carbon.deployment.engine.feature/pom.xml

```diff
@@ -81,6 +84,32 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>org.wso2.carbon.deployment</groupId>
+                                    <artifactId>org.wso2.carbon.deployment.engine</artifactId>
+                                    <version>${carbon.deployment.version}</version>
+                                    <type>bundle</type>
+                                    <overWrite>true</overWrite>
+                                    <outputDirectory>${project.build.directory}/docs</outputDirectory>
+                                    <includes>config-docs/**</includes>
```

Comment:
@kasunbg, this is feature pom configuration. In the component, we save the generated configuration document inside target/classes/ folder. This configuration is to copy generated config-docs in component to the feature.